https://issues.redhat.com/browse/ACM-28735 (#8582)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/import_gui.adoc

You can import existing clusters from the console for each of the available cloud providers.

*Notes:*

> - A hub cluster cannot manage a different hub cluster. You do not have to manually import a hub cluster to be self-managed.

Author: ```suggestion
- A hub cluster cannot manage a different hub cluster. A hub cluster is set up to automatically import and manage itself as a `local-cluster`. You do not have to manually import a hub cluster to manage.
```

> - A hub cluster cannot manage a different hub cluster. A hub cluster is set up to automatically import and manage itself, so you do not have to manually import a hub cluster to manage itself.

Reviewer: import and manage itself as a `local-cluster` (?)

New sentence: You do not have to manually import a hub cluster to be self-managed.

Let's try to avoid "itself"

> - A hub cluster cannot manage a different hub cluster. A hub cluster is set up to automatically import and manage itself, so you do not have to manually import a hub cluster to manage itself.

Author: ```suggestion
- A hub cluster cannot manage a different hub cluster. You do not have to manually import a hub cluster to be self-managed.
```

*Note:* A hub cluster cannot manage a different hub cluster. A hub cluster is set up to automatically import and manage itself, so you do not have to manually import a hub cluster to manage itself.

> The namespace for the imported cluster must match the name of the cluster.

Reviewer 2: I think this should be given an **Important** flag given the trouble users can run into later if they miss this note. See below. Thoughts?

```suggestion
**Important**: The namespace for the imported cluster must match the name of the cluster.
```

Author: Thanks for the suggestion. Instead of adding it as an independent important note, let's add it as a additional note. Having two bold Admonitions back-to-back can be distracting for the user. Let me know what you think

*Notes:* 

- A hub cluster cannot manage a different hub cluster. A hub cluster is set up to automatically import and manage itself, so you do not have to manually import a hub cluster to manage itself.

- The namespace for the imported cluster must match the name of the cluster.

Reviewer: @dockerymick aren't we talking about a local cluster here? Did we review that language here: https://github.com/stolostron/rhacm-docs/blob/2.16_stage/install/adv_config_install.adoc#disablehubselfmanagement

Author: Thanks for the reminder !
